dotcom-rendering/src/components/MatchNav.tsx

```diff
@@ -232,24 +202,35 @@ const YellowBorder = () => (
 );
 
 export const MatchNav = ({ homeTeam, awayTeam, comments }: Props) => (
-	<div>
-		<StretchBackground>
-			<Row>
-				<TeamNav
-					name={homeTeam.name}
-					score={homeTeam.score}
-					crest={homeTeam.crest}
-					scorers={homeTeam.scorers}
-				/>
-				<YellowBorder />
-				<TeamNav
-					name={awayTeam.name}
-					score={awayTeam.score}
-					crest={awayTeam.crest}
-					scorers={awayTeam.scorers}
-				/>
-			</Row>
-			{!!comments && <Comments comments={comments} />}
-		</StretchBackground>
+	<div
+		css={css`
+			display: flex;
+			flex-direction: column;
+			justify-content: space-between;
+			position: relative;
+			padding: ${space[2]}px;
+			background-color: var(--match-nav-background);
```

Comment:
As previous comment, this should be using the `palette` method:

```suggestion
			background-color: ${palette('--match-nav-background')};;
```